.readthedocs.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
```

Comment:
```suggestion
# Just a dummy config file so that readthedocs doesn't fail on security branches.
# This won't result in any docs actually getting built;
# you'll just get a 404 if you try to click on the docs preview link on a PR
version: 2
```